inst/safetyGraphics_app/global.R

```diff
@@ -13,6 +13,10 @@ library(haven)
 
 # create vector of all possible charts
 all_charts <- c("edish","safetyhistogram")
+include_charts <- getShinyOption("charts")
+if (!is.null(include_charts)){
+  all_charts <- all_charts[all_charts %in% include_charts]
```

Comment:
Maybe we print a note to the console here, just in case people forget this option is set. "2 of 5 available charts included being loaded. Set run shinyOptions(charts = NULL) to use all charts.)"